modules/microshift-config-parameters-table.adoc

```diff
@@ -65,10 +65,55 @@ The following table explains {microshift-short} configuration YAML parameters an
 |`number`
 |By default, `etcd` uses as much memory as needed to handle the load on the system. However, in memory constrained systems, it might be preferred or necessary to limit the amount of memory `etcd` can to use at a given time.
 
+|`ingress.defaultHTTPVersion`
+|`number`
+|Determines the default HTTP version to be used for ingress. Default value is `1`, which is the HTTP/1.0 protocol.
+
```

Comment:
Maybe HTTP/1.0 is old, I think it should be HTTP1.1.
@eslutsky

Reply:
yes that is correct by default it using HTTP/1.1 
src:
https://github.com/openshift/router/blob/master/images/router/haproxy/conf/haproxy-config.template#L737